packages/angular/migrations.json

```diff
@@ -1915,21 +1915,6 @@
         }
       }
     },
-    "21.3.5-jest": {
```

Comment:
We cannot change published versions, but with this change, if users migrate to the latest from a version lower than Nx v21.3.5, it will not bump the `jest-preset-angular` package.

I considered adding a migration to lower the version of `jest-preset-angular` (and Jest), but users in this situation are already handling it somehow, and it can get complex to revert some migration changes done for Jest v30.